blivet/devicefactory.py

```diff
@@ -336,7 +337,7 @@ def __init__(self, storage, size, disks, fstype=None, mountpoint=None,
 
         self.child_factory = None
         self.parent_factory = None
-        self.min_luks_entropy = min_luks_entropy
+        luks_data.min_entropy = min_luks_entropy
```

Comment:
I'm not sure this should be set globally. It makes some sense to specify minimal entropy required per device. It makes sense in ``_schedule_partitions()`` and autopart in general, but for "explicitly created" devices it should be possible to specify minimal entropy separately.